content/release-10-18-0/announcements-10-18-0.md

```diff
@@ -144,13 +144,18 @@ All of the impacted injection tokens have been marked as deprecated with 10.18,
 
 The HOOK_SEARCH can be used by developers to extend the web integration of the search in a custom web application. As announced with [release 10.17](/release-10-17-0/announcements-10-17-0), as of a future release, we will refactor the HOOK_SEARCH interface. If you use this interface in a custom-developed UI application, you must migrate to the new version. Details on how to use the new interface will be provided in the Web SDK documentation of the respective release.
 
+##### Deprecation of device-grid model classes, column implementations and services
+
+As announced with [release 10.17](/release-10-17-0/announcements-10-17-0) shared classes, components and services from the device-grid package are deprecated and new ones are created in the core data-grid component.
```

Comment:
Maybe rather mention that common grid API components will be **moved** to `@c8y/ngx-components`, therefore the need for obsolete classes, components and services in `@c8y/ngx-components/device-grid` to be deprecated.

Reply:
Updated, you can take a look again.